README.md

```diff
@@ -193,3 +193,27 @@ file inside your source code repository.
 
     This variable indicates that the asset compilation process will be skipped. Since this only takes place
     when the application is run in the `production` environment, it should only be used when assets are already compiled.
+
+Hot deploy
+---------------------
+In order to dynamically pick up changes made in your application source code, you need to make following steps:
+
+*  **For Ruby on Rails applications**
+
+    Run the built Rails image with the `RAILS_ENV=development` environment variable passed to the [Docker](http://docker.io) `-e` run flag:
+    ```
+    $ docker run -e RAILS_ENV=development -p 8080:8080 rails-hot-deploy
```

Comment:
rails-app (there is nothing special about the image... this suggest that you need to have hot-deploy enabled during the build, which is not the case)